docs/dagmap/dagmap.yaml

```diff
@@ -0,0 +1,187 @@
+meta:
+  version: 1
+style:
+  tags:
+    performance:
+      border: red
+tasks:
+  - label: Track allocations in functions
+    id: track_alloc
+    description: |
+      Maintain a list of allocations inside each function which can be used
+      for freeing things on return, and also for debugging memory usage.
+
+  - label: Catch exceptions
+    id: catch_exceptions
+    description: |
+      Allow exceptions raised in nopython mode to be caught in nopython mode.
+    depends_on:
+      - track_alloc
+
+
+  - label: New IR
+    id: new_ir
+    description: |
+      New intermediate representation for Numba that is backed by a dictionary
+
+  - label: New Type Matching DSL
+    id: type_matching
+    description: |
+      Replace the current DSL for Numba types with something more expressive
+      that can match type patterns
+
+  - label: Declarative type signatures\nfor @overload/@overload_method
+    id: declarative_overload
+    description: |
+      Replace the current DSL for Numba types with something more expressive\n
+      that can match type patterns
+    depends_on:
+      - type_matching
+
+  - label: Rewrite "old-style" implementations
+    id: rewrite_old_impls
+    description: |
+      Rewrite implementations of functions that use the old extension API that
+      separates typing from implementation, and often uses the LLVM builder
+      unnecessarily.
+    depends_on:
+      - declarative_overload
+      - improve_test_suite_tooling
+      - faster_pr_testing
+
+  - label: Unify and add more test suite tooling
+    id: improve_test_suite_tooling
+    description: |
+      Add tools to help with common patterns in testing and unify the ones we have, there's no need for 12 spellings of "is this Python 3"
```

Comment:
numerous long lines in here, if they are wrapped does it mess up the rendering ?

Reply:
It should be fine since I'm using the YAML multi-line string syntax.  Fixed.